src/player.cpp

```diff
@@ -9472,8 +9477,10 @@ void player::do_read( item &book )
         if (reading->intel != 0) {
             add_msg(m_info, _("Requires intelligence of %d to easily read."), reading->intel);
         }
-        if (book_fun_for( book ) != 0) {
-            add_msg(m_info, _("Reading this book affects your morale by %d"), book_fun_for( book ) );
+        //It feels wrong to use a pointer to *this, but I can't find any other player pointers in this method.
+        if( book_fun_for( book, *this ) != 0 )
+        {
```

Comment:
Raise curly bracket one line higher, so it should look like `if( book_fun_for( book, *this ) != 0 ) {`

Reply:
...and done.